.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,12 +1,14 @@
-Please fill in the fields below to submit a pull request.  The more information
-that is provided, the better.
+<!-- Please fill in the fields below to submit a pull request.  The more information
+that is provided, the better. -->
 
-**Fixes issue #**:
+<!-- Insert issue number here. See https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword for more info. -->
```

Comment:
👍  I like the html comment idea! It removes clutter from PRs when submitters forget to delete these lines.